rosidl_typesupport_c/test/test_cli_extension.py

```diff
@@ -14,15 +14,19 @@
 
 import pathlib
 
+from ament_index_python import get_resources
 from rosidl_cli.command.generate.api import generate
 
 TEST_DIR = str(pathlib.Path(__file__).parent)
 
 
 def test_cli_extension_for_smoke(tmp_path):
+    ts = 'c[typesupport_implementations:{}]'.format(
+        list(get_resources('rosidl_typesupport_c'))
+    )
     generate(
         package_name='rosidl_typesupport_c',
         interface_files=[TEST_DIR + ':msg/Test.msg'],
-        typesupports=['c'],
```

Comment:
IIUC correctly `'c'` should still work, so I'd recommend making this two tests - one that checks `'c'` and another that checks `c[...]`

Reply:
See https://github.com/ros2/rosidl_typesupport/pull/112/commits/2e7e8106b22e699ee8704e54b098b4d6ecf9b860.